functoria/lib/functoria_misc.ml

```diff
@@ -72,10 +72,9 @@ module Codegen = struct
 
   let main_ml = ref None
 
-  let generated_header ?(argv=Sys.argv) ?(time=Ptime_clock.now ()) () =
-    Format.asprintf "Generated by %s (%a)."
+  let generated_header ?(argv=Sys.argv) () =
+    Format.asprintf "Generated by '%s'."
```

Comment:
to avoid potential issues, why not use `%S` here, which does a String.escape and surrounds it with `"`?